packages/apps/docs/package.json

```diff
@@ -16,12 +16,8 @@
     "build:scripts": "tsx ./src/scripts/build.ts",
     "build:storybook": "pnpm run build:scripts && storybook build",
     "chromatic": "chromatic -b build:storybook",
-    "cypress:ci": "NEXT_PUBLIC_APP_DEV=test start-server-and-test dev http://localhost:3000 cypress:run",
-    "cypress:local": "NEXT_PUBLIC_APP_DEV=test start-server-and-test dev http://localhost:3000 cypress:open",
```

Comment:
not required anymore?

Reply:
I already removed these on main, I believe.